.github/actions/generate-invocation-ids/generate-ids.py

```diff
@@ -0,0 +1,93 @@
+import json
```

Comment:
Note that `python` is not available on MacOS but `python3` is:
```suggestion
#!/usr/bin/env python3
import json
```